Added temperature sensor code to arduino main sketch

IsabelleTornqvist opened the pull request.

IsabelleTornqvist committed:
Added temperature sensor code to arduino main sketch

timjon commented on SmartCar/Arduino/sketch_main/sketch_main.ino:
```diff
@@ -57,6 +68,36 @@ void setup() {
+  
+  // set the resolution to 9 bit
+    sensors.setResolution(thermometer, 9);
+    sensors.getResolution(thermometer);
```
Is the get really necessary? you are setting the resolution on the line above and you are not using the value for anything

timjon commented on SmartCar/Arduino/sketch_main/sketch_main.ino:
```diff
@@ -57,6 +68,36 @@ void setup() {
     Serial.begin(9600);  
+
+    // Start the sensor
+    sensors.begin();
```
why are you calling something sensors?

timjon commented on SmartCar/Arduino/sketch_main/sketch_main.ino:
```diff
@@ -136,6 +177,10 @@ void loop() {
+
+    //for temperature sensor
+    sensors.requestTemperatures(); // Get temperatures
+    printTemperature(thermometer); //Call 
```
Shouldnt be checked and sent each loop since it might block other information that needs to be sent

IsabelleTornqvist committed:
Added delay when sending temperature.

+Added delay when sending temperature. 
+Allowing for "switching off" sensors, work in progress

IsabelleTornqvist committed:
Added delay when sending temperature.

pontuslae commented on SmartCar/Arduino/sketch_main/sketch_main.ino:
```diff
@@ -1,6 +1,9 @@
 #include <NewPing.h>
```
Replace with 
#include <NewPing.h> // https://bitbucket.org/teckel12/arduino-new-ping/downloads/

IsabelleTornqvist committed:
added a comment to newPing

IsabelleTornqvist committed:
Allow for turning sensors on and off
